code/ui/alivehud/inventorywrapper/inventoryselection/InventorySelection.cs

```diff
@@ -175,28 +175,43 @@ private int NormalizeSlotIndex(int index, int maxIndex)
 
         private int GetKeyboardNumberPressed(InputBuilder input)
         {
-            if (input.Pressed(InputButton.Slot1)) return 1;
-            if (input.Pressed(InputButton.Slot2)) return 2;
-            if (input.Pressed(InputButton.Slot3)) return 3;
-            if (input.Pressed(InputButton.Slot4)) return 4;
-            if (input.Pressed(InputButton.Slot5)) return 5;
+            InputButton[] pressedButton = new[] {
+                InputButton.Slot0,
+                InputButton.Slot1,
+                InputButton.Slot2,
+                InputButton.Slot3,
+                InputButton.Slot4,
+                InputButton.Slot5,
+                InputButton.Slot6,
+                InputButton.Slot7,
+                InputButton.Slot8,
+                InputButton.Slot9
+            };
+
+            for (int i = 0; i < pressedButton.Length; i++)
+            {
+                if (input.Pressed(pressedButton[i]))
+                {
+                    return i;
+                }
+            }
 
-            return 0;
+            return -1;
         }
 
         private static string FormatAmmo(TTTWeapon weapon, Inventory inventory)
         {
             if (weapon.UnlimitedAmmo)
             {
-                return $"{weapon.AmmoClip}";
+                return $"{weapon.AmmoClip} + ∞";
```

Comment:
I tried this before and it didn't look very good.
I prefer having no "+ ∞" after as a single value should be enough to convey that the reserve is infinite.

Reply:
Hmm, no opinion on my end, we probably will just remove infinite ammo at some point anyways. Flip a coin or ask some other people to chime in as a tie breaker.